specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2023-01-01/elasticsan.json

```diff
@@ -901,6 +901,36 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "x-ms-delete-snapshots",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "true",
+              "false"
+            ],
+            "x-ms-enum": {
+              "name": "x-ms-delete-snapshots",
+              "modelAsString": true
+            },
+            "description": "Optional, used to delete snapshots under volume. Allowed value are only true or false."
```

Comment:
With "true", will the base volume be also deleted besides the snapshot?
Please update the description to make it more accurate.
If we need to support both case (delete base volumn or not), we might should add more allowed value. 

Besides that, the description should also include the default value.